### PR TITLE
Replaces HorizontalPicker in the Display Grid sample

### PR DESCRIPTION
**Pull request by qsmmorgan**

### CODE REVIEW

File: arcgis-ios-sdk-samples/Display information/Display grid/DisplayGrid.storyboard [hunk 1]
```diff
@@ -109,31 +109,41 @@
                                             <outlet property="accessoryView" destination="Pbe-fz-3U4" id="UNp-Or-ng3"/>
                                         </connections>
                                     </tableViewCell>
-                                    <tableViewCell clipsSubviews="YES" contentMode="scaleToFill" preservesSuperviewLayoutMargins="YES" selectionStyle="default" indentationWidth="10" textLabel="hyL-D8-7v5" style="IBUITableViewCellStyleDefault" id="ops-Sc-JOY">
+                                    <tableViewCell clipsSubviews="YES" contentMode="scaleToFill" preservesSuperviewLayoutMargins="YES" selectionStyle="default" accessoryType="disclosureIndicator" indentationWidth="10" id="ops-Sc-JOY">
```
[the remark this answers is not in the document]
Comment (qsmmorgan) [reply arcgis-ios-sdk-samples/Display information/Display grid/DisplayGrid.storyboard]: Oh.

File: arcgis-ios-sdk-samples/Display information/Display grid/DisplayGridSettingsViewController.swift [hunk 2]
```diff
@@ -16,92 +16,122 @@
...
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        // Set picker options
-        gridTypePicker.options = gridTypes
-        labelPositionPicker.options = labelPositions
-        labelUnitPicker.options = labelUnits
-        labelFormatPicker.options = labelFormats
+        // Set picker delegates
+        gridColorPicker.delegate = self
```
[the remark this answers is not in the document]
Comment (qsmmorgan) [reply arcgis-ios-sdk-samples/Display information/Display grid/DisplayGridSettingsViewController.swift]: We're getting rid of the color pickers soon anyway so I'm not worried about it.

**Comment by qsmmorgan** [comment]
@philium Ready for re-review.

**Comment by philium** [comment]
Thank you! it looks much better.

**Pull request merged**